edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/EdgeDeploymentCommand.cs

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Commands
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.EdgeDeployment
```

Comment:
Do we want to keep this in "Commands" to be consistent with edgelet and docker projects?

Reply:
I don't think we should dedicate the whole folder for an only command... But I open to suggestions

Reply:
We can always move it back if it confuses others.